Eastern-Research-Group/lew
# #106 Updated the swagger ui to the latest version.

## Related Issues:
* #106 

## Main Changes:
* Fixed an issue with swagger accepting url parameters, by updating the swagger ui to the latest version.
* Removed the sample swagger file: https://lew-dev.app.cloud.gov/api-docs/og_index.html

## Steps To Test:
1. Navigate to http://localhost:9090/api-docs/og_index.html
2. Verify the `Sorry, but this web application is not available at this time.` message is displayed
3. Navigate to http://localhost:9090/api-docs/index.html
4. Verify the lew swagger documentation loads and is displayed
5. Navigate to http://localhost:9090/api-docs/index.html?url=https://raw.githubusercontent.com/z9nsis/swagger/main/test.yaml
6. Verify the lew swagger documentation loads and is displayed
